mirantis/aws-cloud-controller-manager/templates/cloudconfigmap.yaml

```
@@ -0,0 +1,17 @@
+---
+{{- if .Values.cloudConfig.enabled }}
+apiVersion: v1
+data:
+  cloudconfig.cfg: |
+    {{- range $key, $value := .Values.cloudConfig }}
+    {{- if not (eq $key "enabled") }}
+    [{{ camelcase $key }}]
+      {{- range $subKey, $subValue := $value }}
+    {{ $subKey }}={{ $subValue }}
+      {{- end }}
+    {{- end }}
+    {{- end }}
+kind: ConfigMap
+metadata:
+  name: {{- include "aws-cloud-config.name" . | nindent 4 }}
```

Why add this `nindent` here? This makes no sense. You can just:

```suggestion
  name: {{ include "aws-cloud-config.name" . }}
```

Applicable to all other usages of this variable.